apps/api/src/integration/plugins/vercel.integration.ts

```diff
@@ -599,4 +602,79 @@ export class VercelIntegration extends BaseIntegration {
 
     return this.vercel
   }
+
+  private async triggerRedeploy(): Promise<void> {
+    const integration = this.getIntegration<VercelIntegrationMetadata>()
+
+    const projectId: string = integration.metadata.projectId as string
+    const teamId: string = integration.metadata.teamId as string
+
+    if (!projectId || !teamId) {
+      this.logger.warn(
+        '[VERCEL] Missing projectId or teamId, skipping redeploy'
+      )
+      return
+    }
+
+    this.vercel = await this.getVercelClient()
+
+    try {
+      this.logger.log(
+        `[VERCEL] Fetching latest READY deployment for ${projectId}...`
+      )
+
+      const { deployments } = await this.vercel.deployments.getDeployments({
```

Comment:
you shoudldn't check for last deployments. simply create a new deployment using `deployments.createDeployment`. that way i think you woint ned the team id